### PR TITLE
Update documents for more details.

### PR DESCRIPTION
**Pull request by DonnaWuDongxia**

### CODE REVIEW

**Comment by DonnaWuDongxia** [comment]
@halton @kenny-y please help to review.

File: README.md [hunk 1]
```diff
@@ -32,46 +26,13 @@ All supported command-line options of widl-nan can be listed in the help informa
...
-- Buffer, ArrayBuffer and typed array are all supported.
-- Callback and Promise are all supported.
-- Dictionary, Interface, constructor, enum, const and static member are all supported.
-- Overloaded methods and optional arguments are all supported, but method-overload is recommended. 
-- Primitive types are supported except ``` long long```.
-- Stringifier is supported.
-- Event emitter has not been supported directly, JavaScript Wrapper can be used to achive the goal, please refer [this example](https://github.com/otcshare/node-realsense/blob/master/src/slam/index.js#L13).
-
-For more detail or samples, please check the examples under https://github.com/01org/widl-nan/tree/master/test, basically each supported feature has a test case.
+# Getting Start
```
Comment (halton) [inline README.md]: Getting started as Terri suggested.

**Comment by DonnaWuDongxia** [comment]
@halton updated

File: README.md [hunk 2]
```diff
@@ -32,46 +26,13 @@ All supported command-line options of widl-nan can be listed in the help informa
...
-- Buffer, ArrayBuffer and typed array are all supported.
-- Callback and Promise are all supported.
-- Dictionary, Interface, constructor, enum, const and static member are all supported.
-- Overloaded methods and optional arguments are all supported, but method-overload is recommended. 
-- Primitive types are supported except ``` long long```.
-- Stringifier is supported.
-- Event emitter has not been supported directly, JavaScript Wrapper can be used to achive the goal, please refer [this example](https://github.com/otcshare/node-realsense/blob/master/src/slam/index.js#L13).
-
-For more detail or samples, please check the examples under https://github.com/01org/widl-nan/tree/master/test, basically each supported feature has a test case.
+# Getting Started
```
Comment (DonnaWuDongxia) [inline README.md]: Here.

**Pull request merged**